src/components/layout/library-footer.tsx

```diff
@@ -1,6 +1,6 @@
 import Link from "@/components/patterns/elements/drupal-link"
 import {ArrowRightIcon} from "@heroicons/react/20/solid"
-import Lockup from "@/components/patterns/lockup"
+import Lockup from "@/components/patterns/elements/lockup/lockup"
```

Comment:
SUL requested to have the footer lockup updated to match the header for now. We'll circle back to implement the footer redesign during the July 2 release.